### ISSUE TITLE
ipc object domain support

### ISSUE THREAD

**Issue by misson20000** [reswitched/libtransistor#7]

**Comment by misson20000** [comment]
converting a session to a domain does indeed send a session response instead of a domain response:
```
ipcm+0x0  | 04 00 00 00 09 00 00 00  00 00 00 00 00 00 00 00 | ................ |
ipcm+0x10 | 53 46 43 49 00 00 00 00  00 00 00 00 00 00 00 00 | SFCI............ |
ipcm+0x20 | 02 00 00 00 00 00 00 00  00 00 00 00             | ............     |
```
```
sc.ipcMsg(0).setType(5).sendTo("bsd:u").showPacked();
```

**Comment by misson20000** [comment]
closed with e2cf07d0f
